### PR TITLE
Update DG - Edit Command

### PR DESCRIPTION
**Pull request by Chen-Kuei**

### CODE REVIEW

**Review by Weiennn — COMMENTED** [review]
LGTM

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -154,6 +154,28 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
...
 This section describes some noteworthy details on how certain features are implemented.
 
+### Edit Feature
+The `edit` feature allows users to update specific information about certain employees. This feature is supported by 
+three classes, `EditCommand`, `EditCommandParser` and `Model`.
+
+#### Implementation
+
+`EditCommand`: Stores the edited changes to be changed.<br>
+`EditCommandParser`: Parses the user edit command to create an appropriate EditCommand object.<br>
```
Comment (Goh-Li-Ting) [inline docs/DeveloperGuide.md:164]: Should "the user edit command" be "the user's edit command"

**Review by aexolate — APPROVED** [review]
LGTM

**Review by Goh-Li-Ting — APPROVED** [review]
Other than below, LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Chen-Kuei) [reply docs/DeveloperGuide.md]: Ok updated!

**Pull request merged**